core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -635,15 +635,15 @@ public Map<String, Boolean> updateConfiguration(Map<String, String> propertiesMa
           LOG.info("Property {} has been updated to \"{}\" from \"{}\"",
               key.getName(), entry.getValue(), oldValue);
         } else {
-          LOG.debug("Update a non-dynamic property {} is not allowed", key.getName());
+          LOG.warn("Update a non-dynamic property {} is not allowed", key.getName());
```

Comment:
I'm OK with changing the log levels. However, when dynamic conf update is disabled altogether (the default), the log message is not correct. I suggest check `PropertyKey.CONF_DYNAMIC_UPDATE_ENABLED` before `key.isDynamic()` and outside the for loop.